### DISCUSSION TITLE
control motor with raspberry pi and pi4j 2.0

### DISCUSSION THREAD

**Discussion by rezza72** [Pi4J/pi4j#227] [Q&A] [answered]
Hello
I want to control a motor using a Raspberry Pi.
As I searched, all projects are for pi4j v1.0.
I'll be grateful if you guide me.
Thanks in advance.

**Comment by java-boy** [comment]
Hello guys
This is also my problem.
I want to control a servo motor using pi4j 2.0
I need to check a sample if there are any?
Best wishes.

**Comment by FDelporte** [comment, marked as answer]
Within the CrowPi example project, several examples are available for different types of motors, all using V2 of Pi4J.

More info about the project on:

    https://pi4j.com/getting-started/crowpi/

Code is available on:

    https://github.com/Pi4J/pi4j-example-crowpi/blob/main/src/main/java/com/pi4j/crowpi/components/ServoMotorComponent.java
    https://github.com/Pi4J/pi4j-example-crowpi/blob/main/src/main/java/com/pi4j/crowpi/components/StepMotorComponent.java
    https://github.com/Pi4J/pi4j-example-crowpi/blob/main/src/main/java/com/pi4j/crowpi/components/VibrationMotorComponent.java

**Reply by java-boy** [reply]
Thanks a lot @FDelporte ;)

**Comment by FDelporte** [comment]
Will add this to the documentation website asap
https://github.com/Pi4J/pi4j.github.io/issues/24

**Comment by RetoStutz** [comment]
check branch Dev-Arcade on pi4j-example-components

[https://github.com/Pi4J/pi4j-example-components/tree/Dev-Arcade](url)

there is a servo motor with example

check branch Dev-Arcade on pi4j

[https://github.com/Pi4J/pi4j.github.io/tree/Dev-Arcade](url)

there the wiring is explained
